pkg/scalers/stan_scaler.go

```diff
@@ -73,12 +82,12 @@ func NewStanScaler(config *scalersconfig.ScalerConfig) (Scaler, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error parsing stan metadata: %w", err)
 	}
-
+	httpClient := kedautil.CreateHTTPClient(config.GlobalHTTPTimeout, stanMetadata.enableTLS)
```

Comment:
This change disables the CA validation on client side (`InsecureSkipVerify`), is this intended?

Reply:
Hi @JorTurFer,  No, disabling the CA validation on client side is not intended.

Reply:
```suggestion
	httpClient := kedautil.CreateHTTPClient(config.GlobalHTTPTimeout, false)

	if meta.enableTLS {
		config, err := kedautil.NewTLSConfig(meta.cert, meta.key, meta.ca, false)
		if err != nil {
			return nil, err
		}
		httpClient.Transport = kedautil.CreateHTTPTransportWithTLSConfig(config)
	}
```